integration_tests/samples/oauth/oauth_v2.py

```diff
@@ -29,6 +30,10 @@
     scopes=scopes,
     user_scopes=user_scopes,
 )
+redirect_page_renderer = RedirectUriPageRenderer(
```

Comment:
Updated the samples to use the renderer and I've verified the behavior with these two example apps.